app/views/adventures/show.html.erb

```diff
@@ -18,5 +18,10 @@
     backButton: <%= @adventure.back_button %>,
     theme: "<%== escape_javascript @adventure.theme.capitalize %>",
     isOffline: false,
+    <% if @adventure.editable_by?(current_user) %>
+      debuggable: true
+    <% else %>
+      debuggable: false
+    <% end %>
```

Comment:
would this work instead?
```erb
debuggable: <%= @adventure.editable_by?(current_user) %>
```

Reply:
No, it does not. I had it that way originally and it was failing tests.

Reply:
I think basically the `<%= %>` does not work if the contents inside of it result in `false`

Reply:
huh, that's a little surprising. I wonder if it outputs `"false"` as a string instead of a JS value. This is fine as is, was just looking for ways to clean it up a bit.

Reply:
It seems to not output anything at all because the error you get is something like `Unexpected '}'`

Reply:
ahhh, that's likely because you're using `<% ... %>` instead of `<%= ... %>` (notice the `=`).

Reply:
actually, looking at your earlier comment, i'm guessing not. Does `editable_by?` return `false` or `nil` in the non-truthy case?

Reply:
looks like there are ways that that method could return `nil` (eg: if `current_user` is nil). A `!!` in front of this will convert "falsey" values (eg: nil) to an actual `false`.